packages/babel-preset-env/test/targets-parser.spec.js

```diff
@@ -78,6 +78,22 @@ describe("getTargets", () => {
     });
   });
 
+  describe("esmodules", () => {
+    it("returns browsers supporting modules", () => {
```

Comment:
Can we add a test that covers the "override browsers" behavior?

Sidenote: should we warn when a user does esmodules and browsers?

Reply:
Makes sense to add this test. Will do.

Good question on warning: what semantics are the guiding rule for warnings within Babel?

Reply:
Added this test in 15e2672.